app/controllers/targets_controller.rb

```diff
@@ -11,8 +12,18 @@ def show
     render json: @target
   end
 
+  def show_current_user
```

Comment:
I think you have to show targets of the user in the `index` method. If not, any user using this endpoint will see all the targets in the system, wdyt?

Reply:
ok, you're right